migration/v0.26.x-v0.27.x/migrate-swift.sh

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+: "${CK8S_CONFIG_PATH:?"CK8S_CONFIG_PATH is unset"}"
+
+default_common="$CK8S_CONFIG_PATH/defaults/common-config.yaml"
+default_sc="$CK8S_CONFIG_PATH/defaults/sc-config.yaml"
+config_common="$CK8S_CONFIG_PATH/common-config.yaml"
+config_sc="$CK8S_CONFIG_PATH/sc-config.yaml"
+secrets="$CK8S_CONFIG_PATH/secrets.yaml"
+
+configs=( "$default_common" "$default_sc" "$config_common" "$config_sc" )
+
+if [ ! -f "$secrets" ]; then
+    echo "err: $secrets not found" >&2
+    exit 1
+  fi
+
+for config in "${configs[@]}"; do
+  if [ ! -f "$config" ]; then
+    echo "err: $config not found" >&2
+    exit 1
+  fi
+done
+
+merge="$(yq4 ea "explode(.) as \$config ireduce ({}; . * \$config)" "${configs[@]}")"
```

Comment:
Wow, I assume that this basically does the override config thing? That's pretty compact

Reply:
Yeah, I think it should be possible to grab a specific key this way as well by piping the explode and grabbing the key you want before the reduce.